game.rb

```diff
@@ -0,0 +1,128 @@
+require_relative 'dictionary'
+
+class Game
+  attr_reader :word, :guess
+
+  def initialize(word = nil)
+    @dictionary = Dictionary.new.trim_dictionary
+    @word = word || @dictionary.sample(1)[0].split('')
+    @used_letters = []
+    @unused_letters = ('a'..'z').to_a
+    @guess = []
+    @word.each do |_letter|
+      @guess.push('_')
+    end
```

Comment:
Loop in which you update external value, that's a red flag. How about a solution like this:
```
@guess = Array.new(@word.length) { "_" }
```